pack/ms.json

```diff
@@ -672,7 +672,7 @@
     "deck_limit": 3,
     "faction_code": "haas-bioroid",
     "faction_cost": 2,
-    "flavor": "At depths no human tech can reach, a Trieste proxy can manipulate a mindscape with unparalleled precision.\n<champion>Designed by 2019 NISEI World Champion Oliver \"Pinsel\" Siccha</champion>",
+    "flavor": "At depths no human tech can reach, a Trieste proxy can manipulate a mindscape with unparalleled precision.\n<champion><strong>Designed by 2019 NISEI World Champion Oliver \"Pinsel\" Siccha</strong></champion>",
```

Comment:
I thought `<strong>` was implied by `<champion>`.

Reply:
in the current NRDB css, but that's it.  Also, the MS sheet has the bold tags, so the converter brings them through.  I'm inclined to keep this to avoid manual exceptions, but it's worth revisiting the CSS and tidying up the conventions in the dev sheets.

Reply:
Should we remove the champion tag entirely then? Feels like we're double specifying.